build/shell-check.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
I don't see any custom shell scripts in this package, right? If so we can remove shell-check, I assume we don't want to check shell files provided by keycloak itself as we won't be able to fix them here anyway, right?

Reply:
removed